tools/generator/templates/champions/cxx/api.hh.jinja2

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright (c) 2012-2020 Association Prologin <association@prologin.org>
+{% import 'macros/cxx.jinja2' as cxx %}
+
+// This file contains all the API functions for the C++ language, and all the
+// constants, enumerations and structures.
+// It has been generated. You can modify the end of the file if you want, but
+// do not touch the part where constants, enums, structs, and api functions are
+// defined.
+
+#pragma once
+
+#include <string>
+#include <vector>
+{% for constant in game.constant %}
+
+{{ cxx.decl_constant(constant) }}
+{% endfor %}
+{% for enum in game.enum %}
+
+{{ cxx.decl_enum(enum) }}
+{% endfor %}
+{% for struct in game.struct %}
+
+{{ cxx.decl_struct(struct) }}
+{% endfor %}
+{%- for func in game.function %}
+
+{{ func.fct_summary|cxx_comment(doc=True) }}
+{{ func|cxx_prototype }};
+{% endfor %}
+{% for enum in game.enum %}
+
+/// Affiche le contenu d'une enum de type {{ enum.enum_name }}
+void afficher_{{ enum.enum_name }}({{ enum.enum_name }} v);
+{% endfor %}
+{% for struct in game.struct %}
+
+/// Affiche le contenu d'une struct de type {{ struct.str_name }}
+void afficher_{{ struct.str_name }}({{ struct.str_name }} v);
+{% endfor %}
+
+// -----
+// API ends here, you can stop reading now.
+// The rest of the file is generated operators for the structs defined above.
+// You can safely modify them if you want, they are only helpers for you
```

Comment:
Nope, they won't be able to do that when there are symlinks to `/usr/include`

Reply:
I removed this line.